.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
This is a little harsh, I think we should allow for blank issues. 
Most people will follow the template flow, but there'll be times where a blank issue saves time and communicates a point just as effectively.

Reply:
agree

Reply:
This doesn't enforce the template structure, it just means that *something* has to be in the issue body. This is a relatively low bar that significantly increases the quality of communication. If a full issue can exist in a title, this normally relies on there being some other shared context that most readers of the issues do not have.

Reply:
Looks like I'm outvoted. Will remove that from the PR.

For the recond: I strongly believe this is a bad idea.